src/index.html

```diff
@@ -1,13 +1,416 @@
+            <img
+              class="top-bar__logo"
+              src="./images/logo.png"
+              alt="The MET logo"
+            >
+          </a>
+          <div class="top-bar__icons">
+            <a
+              href="tel:+12125357710"
+              class="icon icon--phone">
+            </a>
+            <a
+              href="#menu"
+              class="icon icon--menu">
+            </a>
+          </div>
+        </div>
+        <div class="header__center">
+          <h1 class="header__title">Welcome to The MET</h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="menu__content">
+        <div class="top-bar menu__top">
+          <a href="#home" class="top-bar__logo-link">
+            <img
+              class="top-bar__logo"
+              src="./images/logo.png"
+              alt="The MET logo"
+            >
+          </a>
+          <div class="header__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#home" class="nav__link">Home</a>
+              </li>
+              <li class="nav__item">
+                <a href="#about" class="nav__link">About Us</a>
+              </li>
+              <li class="nav__item">
+                <a href="#now-on-view" class="nav__link">Now on view</a>
+              </li>
+              <li class="nav__item">
+                <a href="#explore" class="nav__link">Explore Online</a>
+              </li>
+              <li class="nav__item">
+                <a href="#contacts" class="nav__link">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="menu__contacts">
+            <div class="menu__contacts__item">
+              <p class="menu__contacts__name">
+                The Met Fifth Avenue
+              </p>
+              <a href="tel:+12125357710" class="menu__contacts__phone">
+                +1 212-535-7710
+              </a>
+            </div>
+            <div>
+              <p class="menu__contacts__name">
+                The Met Cloisters
+              </p>
+              <a href="tel:+12129233700" class="menu__contacts__phone">
+                +1 212-923-3700
+              </a>
+            </div>
+            <a href="tel:+12129233700" class="call-to-order">call to order</a>
+          </div>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about">
+            <div class="about-us__title">
+              <p class="about-us__subtitle">Plan your visit</p>
+              <h2 class="section-title
+                section-title--left
+                section-title--mobile">
```

Comment:
Don`t forget about code style
```suggestion
              <h2
                class="section-title
                section-title--left
                section-title--mobile"
              >
```